### ISSUE TITLE
Expanding some functionality  in config file in post section

### ISSUE THREAD

**Issue by Darktrader** [H0K4/hokachan#3]
Hello, it will be great if you add some extra,  features which can be changed in config file.  (some of them implemented on infinity (8chan) engine). I mean:

1.  Ability to force a subject in OP-post.

2.  Minimum body post lenghth. 

3.  Ability to force image embedding in reply.

Thanks.

**Comment by H0K4** [comment]
yeah I'm working on implementing more stuff from 8chan. problem is that their code no longer gets updated and openIB only works on fixing bugs.

**Comment by Darktrader** [comment]
Hmm... but I saw that czak has some commiting on his [fork](https://github.com/vichan-devel/infinity) of infinity.

**Comment by H0K4** [comment]
1. Don't know if I'm gonna implement that. personally I think a thread subject is redundant in this script most anons just want to post quickly. if is see a commit somewhere Ill add it.
2. i think lainchan has that in their script, I'll probably include it if it doesn't give me too many issues.
3. I will update the post-form.html so that it looks like infinity and you don't need that file selector.js file anymore, i'll take a look once I get to that.
